src/utils/internal/getCallFrame.ts

```diff
@@ -2,25 +2,21 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  try {
-    const inspectionError = new Error()
-    inspectionError.name = 'Inspection Error'
-    throw inspectionError
-  } catch (error) {
-    const frames: string[] = error.stack.split('\n')
+  // Error.stack is available even in IE 10, no need for nullish coalesce
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack#browser_compatibility
+  const frames: string[] = (new Error().stack as string).split('\n')
```

Comment:
Nit, since `stack` is a string (or can be nullable) I prefer the non-null assertion operator instead of a cast.

```suggestion
  const frames: string[] = new Error().stack!.split('\n')
```

Reply:
done